apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -244,6 +244,30 @@ private class CredentialWrapper : ITokenAccess
         }
 
         [Fact]
+        public async Task ReadAllowsNewApis()
+        {
+            // ReSharper disable once RedundantAssignment
+            int rowsRead = -1;
```

Comment:
Is this set to -1 here and then 0 below to distinguish different kinds of failures by looking at the test error? Perhaps a string or enum describing the 3 possible states would be better then.